recipe/meta.yaml

```diff
@@ -1,34 +1,29 @@
-# Using the name variable with the URL in line 13 is conviniet
-# when copying and pasting from another recipe, but not really needed.
-{% set name = "fluidsynth" %}
 {% set version = "2.2.8" %}
 
 package:
-  name: {{ name|lower }}
+  name: fluidsynth
   version: {{ version }}
 
 source:
-  url: https://github.com/FluidSynth/{{ name }}/archive/v{{ version }}.tar.gz
+  url: https://github.com/FluidSynth/fluidsynth/archive/v{{ version }}.tar.gz
   sha256: 7c29a5cb7a2755c8012d941d1335da7bda957bbb0a86b7c59215d26773bb51fe
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage("fluidsynth", max_pin="x.x") }}
```

Comment:
I asked https://abi-laboratory.pro to add fluidsynth, but it looks like minor is the right level from looking at their release notes.